Update grant submission guide to reflect changes using the new Colony… (#1259)
urbit/urbit.org: content/grant-submission-guide.md

+++
title = "Guide to Creating a Grant Proposal"
date = "2021-11-25"
weight = 1
description = "A comprehensive guide to creating a grant proposal"
+++

> The Urbit project relies on our community to help realize its potential. While the Foundation and the Community specifies the direction of development through bounties, we encourage anyone with an idea about how to improve Urbit to submit a grant proposal. This guide will describe the different types of proposals, the requested format, and the entire workflow for a successful project. If you want some ideas about projects that have a good lilelyhood of acceptance, take a look at the Grant Ideas notebook in the Foundation group.

Reviewer: ```suggestion
The Urbit project relies on our community to help realize its potential. While the Foundation and the Community specifies the direction of development through bounties, we encourage anyone with an idea about how to improve Urbit to submit a grant proposal. This guide will describe the different types of proposals, the requested format, and the entire workflow for a successful project. If you want some ideas about projects that have a good likelihood of acceptance, take a look at the Grant Ideas notebook in the Foundation group.
```

### The Colony

> The Colony consists of invited community members that have proven track record of completing significant projects. Members of the Colony will be available to "Champion" (manage and mentor) proposals. A proposal is accepted once a Champion adopts it. The Champion will then be part of the team with the Contributor that sees the proposal through to completion.

Reviewer: ```suggestion
The Colony consists of invited community members that have a proven track record of completing significant projects. Members of the Colony will be available to *champion* (manage and mentor) proposals. A proposal is accepted once a Colony member champions it. The Champion will then work with the contributor to see the proposal through to completion.
```

> There will also be members of the Colony thar are invited to be on the Oracle team. The Oracle team is responsibile for ensuring proposals are correctly stuctured and payments meet current norms.

Reviewer: I'd remove this line and reference to oracles in general.

### Types of Proposals

While we will entertain just about any idea that comes from the community, we are particularly interested in "capital goods":

> projects that provide new capabilities to the widest number of users or developers or add tooling to the platform.

Reviewer: ```suggestion
projects that provide new capabilities to the widest number of users and/or developers.
```

- Design Work - design work for the urbit.org website
- Documentation - writing guides or website content
- Community - organizing events and meetups, staffing the help desk

> - Apprenticeship - smaller scope coding projects where the empahsis is on deveolping skills

Reviewer: ```suggestion
- Apprenticeship - smaller scope coding projects where the emphasis is on developing skills
```

We have need for lots of different skill sets &mdash; not just coding.

> ### Proposal Format

Reviewer: ```suggestion
### Format
```

Let's move this section to after the Submission and Workflow section.

Your proposal should include the following sections:

- Project name
- Brief description of project (short blurb for the thumbnail)
- Project Overview
- Project Scope
- Detailed behavior description
- Detailed design description
- Your experience and why you are the right person for this project

> - Milestones:

Reviewer: ```suggestion
- Milestones: incremental deliverables
```

>   - Milestones are sub-goals of project (if any)

Reviewer: ```suggestion
```

>   - Specific deliverables

Reviewer: ```suggestion
  - What will be delivered
```

>   - Expected delivery dates

Reviewer: ```suggestion
  - Estimated completion date
```

>   - Expected compensation in USD

Reviewer: ```suggestion
  - Requested compensation in USD
```

For some good examples, take a look at these proposals to see the level of detail we are looking for:

- [WebRTC](https://urbit.org/grants/webrtc-gall-agent-and-external-app)
- [Urbit Visor](https://urbit.org/grants/login-with-urbit)
- [Urbit Meetup Bill of Materials](https://urbit.org/grants/meetup-bom)

> ### Proposal Submission and Project Workflow

Reviewer: ```suggestion
### Submission and Workflow
```

> Once you have your proposal draft, you will need to get it Championed by a member of the Colony. Start by creating a group on your ship named after your proposal. In this group, create a Proposal Notebook and a Discussion channel. Create a notebook entry for each of the proposal areas noted above. Notify the Foundation of your proposal by filling out the **[Propose a Project](https://airtable.com/shrgd9pQliamOFWvY)** form.

Reviewer: This should be a numbered list of steps.

From this point, your project will go through these stages:

> - **Draft**: Your proposal will be advertised to the Colony. Interested members will review it, ask questions, and offer feedback. If acceptable, a member will become your Champion. Your Champion will work with you to polish you proposal, refine milestones (intermediate deliverables along with payments), create the proposal markdown, then create the Pull Request against the urbit.org website

Reviewer: Would be good to indicate that there's no action required on your part to advertise this to the Colony.

```suggestion
- **Draft**: Your proposal will be advertised to the Colony. Interested members will review it, ask questions, and offer feedback. If acceptable, a member will offer to champion your project. Your Champion will work with you to polish you proposal, refine milestones (intermediate deliverables along with payments), and then create the Pull Request against the urbit.org website.
```

> - **Approval**: Once the proposal PR is created, the Oracle team will review for approval.

Reviewer: ```suggestion
- **Approval**: Once the proposal PR is created, the Foundation team will review the proposed compensation, and either sign off on the proposed amount or propose changes.
```

- **Contract**: Once your proposal is approved, the PR is merged and a contract is sent for signature.
- **Live**: When the contract is signed, you can start work on your project.
- **Milestone**: When you complete a milestone (or the entire project), you will create an invoice an submit it using the

>   [Submit Invoice](https://airtable.com/shrXXCs1uaxtNSBcg) form. In order to submit an invoice, you will need review and approval from your Champion and two other Colony members.

Reviewer: ```suggestion
  [Submit Invoice](https://airtable.com/shrXXCs1uaxtNSBcg) form. In order to submit an invoice, you will need review and approval from your Champion and two other Colony members (your Champion will ensure that this happens).
```
